tests/integration/targets/role_version_update_single_node/tasks/main.yml

```diff
@@ -1,8 +1,8 @@
 ---
 - environment:
     SC_HOST: "{{ sc_host }}"
-    SC_USERNAME: "{{ sc_config[sc_host].sc_username }}"
-    SC_PASSWORD: "{{ sc_config[sc_host].sc_password }}"
+    SC_USERNAME: "{{ sc_username_50 }}"
+    SC_PASSWORD: "{{ sc_password_50 }}"
```

Comment:
I will revert that change, it would prevent testing role on different clusters. For CI tests there is idea to select with  `sc_host` the test cluster, then all other relevant data/configuration needed for CI test are in `integration_config.yml`, at `sc_config[sc_host].xyz`.

I believe you wanted to add a whole additional host to your `integration_config.yml` (one of out test NUCs). You can do that, we intentionally commit to git only `integration_config.yml.j2`, so we can have extra hosts in `integration_config.yml`.